management/src/main/java/nearsoft/skytouch/management/producer/ProductProducer.java

```diff
@@ -0,0 +1,8 @@
+package nearsoft.skytouch.management.producer;
+
+import nearsoft.skytouch.management.channel.ProductChannel;
+import org.springframework.cloud.stream.annotation.EnableBinding;
+
+@EnableBinding(ProductChannel.class)
+public class ProductProducer {
```

Comment:
What is this class for?

Reply:
This class is needed to make use of the ProductChannel class